src/Shared/Components/Security/Vulnerabilities/Vulnerabilities.tsx

```diff
@@ -32,19 +32,15 @@ const Vulnerabilities = ({
     environmentId,
     setVulnerabilityCount,
     SecurityModalSidebar,
-    getSecurityScan,
 }: VulnerabilitiesProps) => {
-    const isScanV2Enabled = window._env_.ENABLE_RESOURCE_SCAN_V2
-    const { scanDetailsLoading, scanResultResponse, scanDetailsError, reloadScanDetails } =
-        useGetSecurityVulnerabilities({
-            appId: String(applicationId),
-            artifactId: String(artifactId),
-            envId: environmentId,
-            isScanEnabled,
-            isScanned,
-            isScanV2Enabled,
-            getSecurityScan,
-        })
+    const [scanResultLoading, scanResultResponse, scanResultError, reloadScanResult] = useAsync(
+        () => getSecurityScan({ artifactId, appId: applicationId }),
+        [],
```

Comment:
Do we need to add dep?

Reply:
not req, in a map call will be different for each material